_docs/_latest/develop/branch-methodology.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Branch Methodology
+order: 003
+---
+
+#  {{ page.title }}
+
+This page describes the branches in the Forseti repo and how they're used
+and managed.
+
+## Branches
+
+### General Availability (GA) Branches
+
+GA branches are the main branches you'll use to inspect the currently stable &
+production-ready codebase, create a Pull Request (PR), or deploy to production.
+
+* `dev`: development branch
+* `master`: latest stable release; suitable for deploy to production
+* `forsetisecurity.org`: documentations for Forseti website.
+
+The `dev` branch is the starting point where you can create a new PR,
+and where the PR will be merged into the codebase after code review. Although
+there are unit tests, the development branch is still considered to be unstable
+because functional and system tests are not yet complete.
+
+The `master` branch is checkpointed code from the `dev` branch that
+has passed functional and system tests. The `master` branch is considered
+to be stable and suitable for production-use.
+
+### Next-Generation Branches
+
+Branches will be also be created, to contain the development for the
+next-generation of Forseti.  These next-generation branches will be prefixed
+with a version number.
+
+Examples of the next-generation `N.0-<branches>`:
+
+* `2.0-dev`: development branch
```

Comment:
Nice, I think you should drop the explanation after the `;` as that might not always be the case for what's in those branches.

Reply:
I'd also turn this section into an explanation of N.N-EAP, N.N-RC, N.N-dev, N.N-master...

Typically the team will begin with a `N.N-dev` branch the once the the code reaches a point for Early Access the team will create and announce an eap branch, e.g. `N.N-eap1`. Upon completion of an EAP phase the team will create Release Candidates and encourage testing, e.g. `N.N-rc1`. Once all testing requirements are satisfied the team will merge the completed code into `N.N-dev` then into `N.N-master`, then into `dev`, and finally `master`.

An example of this workflow is described below for version 2.0:
* `2.0-dev`: A daily development branch.
* `2.0-eap1`: A branch used for Early Access testing.
* `2.0-rc1`: A possible release candidate.
* `2.0-rc2`: A second release candidate with additional changes.
* `2.0-master`: latest stable release; suitable for deploy to production

Reply:
Awesome suggestion... done and done.